.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -3,7 +3,7 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: 'Type: Bug'
```

Comment:
:question: Why are these changes necessary ? :thinking:

Reply:
They are not, but I just copied over the .github folder from main :sweat_smile: I can remove it of course but I think I'd prefer to just keep everything as close to main as possible